src/utils/utils.js

```diff
@@ -2,6 +2,11 @@
 
 angular.module('obiba.mica.utils', ['schemaForm'])
 
+  .factory('urlencode', function() {
```

Comment:
urlencode => urlEncode

and you probably have to submit a similar PR for branch-1.3.x